### PR TITLE
fix(playground): checkbox bug

### PR DESCRIPTION
**Pull request by NicholasLYang**
## Summary
Fixes bug in playground where changing the typescript checkbox deactivates the jsx one.

### CODE REVIEW

**Comment by ematipico** [comment]
That is intended. Script snippet don't allow JSX nor Typescript

**Comment by MichaReiser** [comment]
> That is intended. Script snippets don't allow JSX nor Typescript

Why so?

**Comment by MichaReiser** [comment]
> > > That is intended. Script snippets don't allow JSX nor Typescript
> > 
> > 
> > Why so?
> 
> A script can't have JSX because it requires a compilation step. TypeScript can't be loaded as a script because it requires a compilation step. Is there something I am missing?

The only difference between `SourceType::Script` and `Module` is that 
* `Module` is in strict mode by default
* `Module` allows ES `import`/`export` statements. 

The `Script` and `Module` wording matches the wording of the ECMAScript specification ([Script](https://tc39.es/ecma262/multipage/ecmascript-language-scripts-and-modules.html#prod-Script), [Module](https://tc39.es/ecma262/multipage/ecmascript-language-scripts-and-modules.html#sec-modules)

**Comment by NicholasLYang** [comment]
If we didn't want script snippets to have TypeScript or JSX, perhaps the better option would be to hide the checkboxes when we're on script mode? Or deactivate them. But I don't see anything wrong with someone trying script + jsx + typescript

**Review by ematipico — APPROVED** [review]

File: website/playground/src/SourceTypeSelect.tsx [hunk 1]
```diff
@@ -51,8 +51,6 @@ export default function SourceTypeSelect(
 							type="checkbox"
 							checked={isTypeScript}
 							onChange={(e) => {
-								setIsTypeScript(false);
-								setIsJsx(false);
```
Comment (ematipico) [inline website/playground/src/SourceTypeSelect.tsx:55]: Yeah this was an oversight of mine. I wanted to apply these two calls when source type changes. Sorry about that

**Pull request merged**